ResNetD (#16162)
huggingface/transformers: docs/source/model_doc/resnetd.mdx

<!--Copyright 2022 The HuggingFace Team. All rights reserved.

Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
the License. You may obtain a copy of the License at

http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
specific language governing permissions and limitations under the License.
-->

> # resnetd

Reviewer: ```suggestion
# ResNet-D
```

## Overview

> The resnetd model was proposed in [Bag of Tricks for Image Classification with Convolutional Neural Networks](https://arxiv.org/abs/1812.01187) by Tong He, Zhi Zhang, Hang Zhang, Zhongyue Zhang, Junyuan Xie, Mu Li.

Reviewer 2: ```suggestion
The ResNet-D model was proposed in [Bag of Tricks for Image Classification with Convolutional Neural Networks](https://arxiv.org/abs/1812.01187) by Tong He, Zhi Zhang, Hang Zhang, Zhongyue Zhang, Junyuan Xie, Mu Li.
```

> ResNetD is a modification on the ResNet architecture. It utilises an embedding layer (stem) composed by three `3x3` convolution layers  and an average pooling in the shortcut to downsample the input.

Reviewer: ```suggestion
ResNet-D is a modification on the ResNet architecture. It utilises an embedding layer (stem) composed by three `3x3` convolution layers  and an average pooling in the shortcut to downsample the input.
```

> ResNet-D is a modification on the ResNet architecture. It utilises an embedding layer (stem) composed by three `3x3` convolution layers  and an average pooling in the shortcut to downsample the input.

Reviewer 2: ```suggestion
ResNet-D is a modification of the ResNet architecture. It utilises an embedding layer (stem) composed by three `3x3` convolution layers  and an average pooling in the shortcut to downsample the input.
```
